src/patternfly/layouts/Split/styles.scss

```diff
@@ -1,16 +1,27 @@
 @import "../../patternfly-utilities";
 
 .pf-l-split {
+  $pf-l-split__item--Padding: 0 !default;
+  $pf-l-split__item--PaddingLeft--first-child: 0 !default;
+  $pf-l-split__item--PaddingRight--last-child: 0 !default;
   display: flex;
   padding: 0;
   margin: 0;
 
-  > :nth-child(2),
-  > .pf-l-split__primary {
-    flex: 1;
+  & > &__item.pf-is-primary {
```

Comment:
In my opinion, once we start giving flex width values it means that this should be a grid instead.
But why do we need those values if we just want it to be as large as it's content? or what am I missing?

plus if we make it a grid we can use a grid gap instead of a padding.

Reply:
thanks a lot!